app/workers/remove_assignments_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
File name and class name should be singular. (This is how they are called, so it's causing the tests to fail.)